server/lib/house/house.create.js

```diff
@@ -15,7 +16,9 @@ const db = require('../../models');
  * });
  */
 async function create(house) {
-  return db.House.create(house);
+  const newHouse = await db.House.create(house);
+  this.event.emit(EVENTS.HOUSE.CREATE);
```

Comment:
maybe we want to send the newHouse here?

```
this.event.emit(EVENTS.HOUSE.CREATED, newHouse.get({ plain: true}));
```

Even if it's not useful in this PR, I think it make sense for future uses